lib/web/fotorama/fotorama.js

```diff
@@ -1140,12 +1140,24 @@ fotoramaVersion = '4.6.4';
 
     function addEvent(el, e, fn, bool) {
         if (!e) return;
-        const options = {
-            get passive() {
-                return e === 'wheel';
-            }
-        }
-        el.addEventListener ? el.addEventListener(e, fn, options) : el.attachEvent('on' + e, fn);
+        let supportsPassive = false;
+        try {
+            const options = {
+                get passive() {
+                    supportsPassive = true;
+                },
+            };
+
+            document.addEventListener('test', null, options);
+            document.removeEventListener('test', null, options);
+        } catch (err) {}
```

Comment:
This feature detection is not dependant on any of the arguments to this function (nor any inherited context). Can we move this somewhere else to improve performance / reduce the amount of duplicate work the browser needs to do to detect the feature? I don't know how often / how many times this function (`addEvent()`) is called.